### PR TITLE
RFC - Attempt at making deployment docs easier to follow

### PR DESCRIPTION
**Pull request by j-lea**
* A short explanation of the proposed change:
Remove the link to each specific platform from the main readme, and instead link to the top of the deployment readme. 

This is because people often miss the first instructions (download the package.zip from the releases page) as it is hidden in pre requisites at the top of the page.

This is a move to making it simpler, but I am not sure that it is the best way. Other options include:

- Replicating the  pre-requisites in each platform-specific section in one page
- Having a different readme page for each platform, and replicating the pre-requisites in each of those

* [X] I have reviewed the [contributing guide](https://github.com/pivotal/postfacto/blob/master/CONTRIBUTING.md)

* [X] I have made this pull request to the `master` branch

* [X] I have run all the tests using `./test.sh`.

* [X] I have added the [copyright headers](https://github.com/pivotal/postfacto/blob/master/license-header.txt) to each new file added

* [ X] I have given myself credit in the [humans.txt](https://github.com/pivotal/postfacto/blob/master/humans.txt) file (assuming I want to)

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-  * [Cloud Foundry](#cloud-foundry)
-  * [Heroku](#heroku)
-  * [Configuration](#configuration)
-  * [Testing your deployment](#testing-your-deployment)
-
-## Prerequisites
 1. Download and extract the latest package from the [releases page](https://github.com/pivotal/postfacto/releases)
-2. Choose a name for your app, we'll refer to this as `app-name` from now on
+1. Choose a name for your app, we'll refer to this as `app-name` from now on
+1. [Configure your deployment](#configuration) by adding Google Auth, enabling analytics, increasing the session timeout or setting the environment variable to run without Redis (only for v4.3.0 or higher with Postgres)
```
Comment (textbook) [inline deployment/README.md:5]: Does it make sense to move this whole section up, given its position in this list?

**Review by textbook — APPROVED** [review]
I think this is a good first step, hopefully it guides people a bit more through the process rather than encouraging jumping straight to the specific platform and missing the prerequisites.

**Comment by textbook** [comment]
Should probably rebase on top of the changes from #305 and bring that into the TOC too

File: deployment/README.md (hunk 1 above)
Comment (j-lea) [reply deployment/README.md:5]: What do you mean? Move the configuration section up above the specific platform instructions?

Comment (j-lea) [reply deployment/README.md:5]: ^ If yes, I agree. Might moving the platform instructions down might also make them less tempting to click on

Comment (textbook) [reply deployment/README.md:5]: Yes, exactly; to reflect the fact that point 3 is the configuration, point 4 is the platforms and point 5 is testing the deployments, the rest of the docs should follow that order to make it more intuitive to navigate.

**Comment by j-lea** [comment]
@textbook have rebased, moved config up and linked directly to the config points at the top. Do you think having two bullet point lists of links looks weird?

**Comment by textbook** [comment]
I like it! @liamdarmody what are your thoughts on how this flows?

Full preview: https://github.com/pivotal/postfacto/blob/deployment-docs-reshuffle/deployment/README.md

**Comment by jmichelgarcia** [comment]
Merge ?

**Comment by j-lea** [comment]
> Merge ?

I dont have write access anymore :( I am not allowed

**Pull request merged**